packages/data-context/src/data/ProjectConfigManager.ts

```diff
@@ -191,12 +191,6 @@ export class ProjectConfigManager {
       return
     }
 
-    const result = await isDependencyInstalled(bundler, this.options.projectRoot)
```

Comment:
`await framework.dependencies(...)` will return all the deps __including__ the bundler (if required) so no need to duplicate the bundler check.